public/docs/moduleinfo.php

```diff
@@ -50,7 +51,17 @@
 
                         <?php //require_once '../inc/notice_work_in_progress.php'; ?>
 
-                        <p>Jedes Modul, das mit dem MMLC funktionieren soll brauch eine <code>moduleinfo.json</code> Datei. In dieser Datei werden die wichtigsten Angaben und Einstellungen für den MMLC zum Modul konfiguriert. In diesem Abschnitt werden die Felder und dessen Funktionen in der <code>moduleinfo.json</code> beschrieben.</p>
+                        <p>Jedes Modul, das mit dem MMLC funktionieren soll braucht eine <code>moduleinfo.json</code> Datei. In dieser Datei werden die wichtigsten Angaben und Einstellungen für den MMLC zum Modul konfiguriert. In diesem Abschnitt werden die Felder und dessen Funktionen in der <code>moduleinfo.json</code> beschrieben.</p>
+                    </section>
+
+                    <section>
+                        <h3 id="example">
+                            Beispiel
+                            <a href="#example" class="anchor">#</a>
+                        </h3>
+
+                        <p>Eine Vollständige <code>moduleinfo.json</code> könnte wie folgt aussehen.</p>
+                        <code class="block"><?= file_get_contents('../../examples/full-example.json'); ?></code>
```

Comment:
Get the latest commit and look in docs/tutorial.php. 
Look vor `<?= loadExample2('/tutorial_010.json', 'json'); ?>`. You should also use `<= loadExample2(FILENAME, FILETYPE); =>`